framework/components/ecl-carousels/ecl-carousels.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * Carousel
+ * @define carousel ; weak
+ */
+
+.ecl-carousel {
+  align-items: stretch;
+  background-color: #000;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
```

Comment:
Approach changed a bit in the meanwhile, but I used the instructions in latest push.